client/src/pages/Feed.js

```diff
@@ -1,37 +1,189 @@
 import Posts from "components/Feed/Posts";
-import { optionsReducer, feedReducer } from "hooks/reducers/feedReducers";
+import { optionsReducer, feedReducer, postsReducer } from "hooks/reducers/feedReducers";
+
+// Constants
+import { theme, mq } from "constants/theme";
+import { BLACK, DARKER_GRAY, ROYAL_BLUE, WHITE } from "constants/colors";
 import {
   ADD_OPTION,
   REMOVE_OPTION,
   REMOVE_ALL_OPTIONS,
   TOGGLE_STATE,
   SET_VALUE,
+  SET_POSTS,
 } from "hooks/actions/feedActions";
 
 // ICONS
 import SvgIcon from "components/Icon/SvgIcon";
 import creatPost from "assets/icons/create-post.svg";
+import { ReactComponent as FiltersIcon } from "assets/icons/filters.svg";
 
 export const FeedContext = React.createContext();
 
+const { Content, Sider } = Layout;
+
+// feed types
+const HELP_TYPE = {
+  ALL: 'All posts',
+  REQUEST: 'Requesting help',
+  OFFER: 'Offering help'
+};
+
 const initialState = {
+  selectedType: '',
+  showFilters: false,
   filterModal: false,
   createPostModal: false,
   activePanel: null,
-  location: "",
+  location: '',
 };
 
+const SiderWrapper = styled(Sider)`
+  background-color: ${WHITE};
+  height: calc(100vh - 6rem);
+  overflow-x: hidden;
+  padding-top: 3.3rem;
+  position: fixed;
+
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    display: none;
+  }
+`;
+
+const FiltersWrapper = styled.div`
+  border-top: 0.05rem solid rgba(0, 0, 0, 0.5);
+  margin: 0 2rem;
+  padding-top: 2rem;
+
+  button {
+    align-items: center;
+    background-color: transparent;
+    border: none;
+    color: ${BLACK};
+    cursor: pointer;
+    display: flex;
+    font-family: ${theme.typography.font.family.display};
+    font-size: ${theme.typography.size.large};
+    font-weight: bold;
+    margin-bottom: 1rem;
+    padding: 0;
+
+    span {
+      align-items: center;
+      border: 0.1rem solid ${ROYAL_BLUE};
+      border-radius: 50%;
+      color: ${ROYAL_BLUE};
+      display: flex;
+      height: 4.2rem;
+      justify-content: center;
+      margin-right: 1rem;
+      width: 4.2rem;
+
+      svg {
+        fill: ${ROYAL_BLUE};
+        height: 2rem;
+        width: 2rem;
+      }
+    }
+  }
+`;
+
+const MenuWrapper = styled(Menu)`
+  &.ant-menu {
+    .ant-menu-item {
+      border-left: 0.5rem solid ${WHITE};
+      color: ${DARKER_GRAY};
+      font-size: ${theme.typography.size.large};
+
+      &:hover {
+        color: ${ROYAL_BLUE};
+      }
+    }
+
+    .ant-menu-item-selected {
+      background-color: transparent;
+      border-left: 0.5rem solid ${ROYAL_BLUE};
+      color: ${ROYAL_BLUE};
+      font-weight: bold;
+    }
+  }
+`;
+
+const LayoutWrapper = styled(Layout)`
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    background-color: ${WHITE};
+  }
+
+  @media screen and (min-width: ${mq.tablet.narrow.minWidth}) {
+    background-color: #f2f2f2;
+
+    .create-post,
+    .filter-box {
+      display: none;
+    }
+  }
+`;
+
+const ContentWrapper = styled(Content)`
```

Comment:
I think the height of the content is dependent on how many posts there are right now. Is it possible to have the height not change?

![Screen Shot 2020-05-11 at 5 19 09 PM](https://user-images.githubusercontent.com/42228386/81624753-d0344a80-93ab-11ea-9b75-3e75dfd35dbe.png)
![Screen Shot 2020-05-11 at 5 19 17 PM](https://user-images.githubusercontent.com/42228386/81624763-d3c7d180-93ab-11ea-96a0-c814db2d0789.png)

Reply:
Which rule are you referring to?

Reply:
What do you mean by rule?

Reply:
ContentWrapper doesn't have a height rule. Not sure about which height is changing.

Reply:
This is what I mean by the changing height:
![height](https://user-images.githubusercontent.com/42228386/81642251-d6402080-93d7-11ea-87f7-ba49d0e96e19.gif)

Reply:
Done.